.eslintrc

```diff
@@ -16,7 +16,7 @@
   "plugins": ["react", "@typescript-eslint"],
   "ignorePatterns": ["/root/webpack.config.js"],
   "rules": {
-    "no-console": "error",
+    "no-console": ["error"],
```

Comment:
Is there any difference when you make it an array.

Reply:
1. The issue was that the `eslint` was being initialized to run on the files that are located in the parent directory, not traversing the whole directory tree. Reason for many affected files.

2. Making it array ensures that we are rasing the error but give the flexibility to allow some consoles like `console.warn` which can be used outside debugging but to output some information intended for some users (Though I see rare cases of us using these)